src/Microsoft.Sbom.DotNetTool/ConsolidationService.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is there any difference here vs the other ConsiolidationService.cs added in the previous PR? If not, can we just [reference that one in this proj similarly to how we reference program.cs](https://github.com/microsoft/sbom-tool/blob/main/src/Microsoft.Sbom.DotNetTool/Microsoft.Sbom.DotNetTool.csproj#L12)?

Reply:
The ones in the `Microsoft.SbomTool` project all include `Environment.Exit` at the end of the task, while none of the ones in the `Microsoft.DotNetTool` project have those. I think the exits were added in a later PR and I'm hesitant to change the existing pattern without fully understanding how it would impact the E2E